README.md

```diff
@@ -30,8 +30,10 @@ Supports these build/install tools:
 
 <!-- markdownlint-disable MD013 -->
 
-| Variable Name | Required | Default | Description                               |
-| ------------- | -------- | ------- | ----------------------------------------- |
-| NODE_VERSION  | False    |         | Node.js version to use for build/install  |
-| BUILD_TOOL    | False    |         | Tool used to perform the build [npm/yarn] |
-| PATH_PREFIX   | False    |         | Path/directory to Node.js project code    |
+| Variable Name | Required | Default    | Description                               |
+| ------------- | -------- | ---------- | ----------------------------------------- |
+| NODE_VERSION  | False    |            | Node.js version to use for build/install  |
+| BUILD_TOOL    | False    |            | Tool used to perform the build [npm/yarn] |
```

Comment:
Shouldn't this be required? I don't see any default in the code and looking over the flow one of them needs to be defined for it to do anything.

Reply:
Nevermind, I see you fix this in #4